Latest build wont start in docker

just did a git pull and built the docker image, it starts up but crashes after about 5 seconds.
docker logs show this:

```
[03:53:51] [INF] Jellyfin version: 10.2.2.0
[03:53:51] [INF] Arguments: ["/jellyfin/jellyfin.dll", "--datadir", "/config", "--cachedir", "/cache", "--ffmpeg", "/usr/local/bin/ffmpeg"]
[03:53:51] [INF] Operating system: Linux 4.9.0.8
[03:53:51] [INF] Architecture: X64
[03:53:51] [INF] 64-Bit Process: True
[03:53:51] [INF] User Interactive: True
[03:53:51] [INF] Processor count: 8
[03:53:51] [INF] Program data path: /config
[03:53:51] [INF] Web resources path: /jellyfin/jellyfin-web/src
[03:53:51] [INF] Application directory: /jellyfin/
[03:53:51] [INF] Setting cache path to /cache
[03:53:51] [INF] Loading assemblies
[03:53:52] [INF] Sqlite version: 3.26.0
[03:53:52] [INF] Sqlite compiler options: COMPILER=gcc-5.4.0 20160609,DEFAULT_FOREIGN_KEYS,ENABLE_COLUMN_METADATA,ENABLE_FTS3_PARENTHESIS,ENABLE_FTS4,ENABLE_FTS5,ENABLE_JSON1,ENABLE_RTREE,THREADSAFE=1
[03:53:52] [INF] Default journal_mode for /config/data/users.db is wal
[03:53:52] [INF] PRAGMA synchronous=1
[03:53:52] [INF] Default journal_mode for /config/data/authentication.db is wal
[03:53:52] [INF] PRAGMA synchronous=1
[03:53:52] [INF] Default journal_mode for /config/data/activitylog.db is wal
[03:53:52] [INF] PRAGMA synchronous=1
[03:53:52] [INF] Default journal_mode for /config/data/displaypreferences.db is wal
[03:53:52] [INF] PRAGMA synchronous=1
[03:53:52] [INF] Disposing CoreAppHost
[03:53:52] [FTL] Unhandled Exception
System.NullReferenceException: Object reference not set to an instance of an object.
   at Emby.Server.Implementations.ApplicationHost.SetStaticProperties()
   at Emby.Server.Implementations.ApplicationHost.RegisterResources(IServiceCollection serviceCollection)
   at Emby.Server.Implementations.ApplicationHost.InitAsync(IServiceCollection serviceCollection)
   at Jellyfin.Server.Program.StartApp(StartupOptions options)
   at Jellyfin.Server.Program.Main(String[] args)
   at Jellyfin.Server.Program.<Main>(String[] args)

Unhandled Exception: System.NullReferenceException: Object reference not set to an instance of an object.
   at Emby.Server.Implementations.ApplicationHost.SetStaticProperties()
   at Emby.Server.Implementations.ApplicationHost.RegisterResources(IServiceCollection serviceCollection)
   at Emby.Server.Implementations.ApplicationHost.InitAsync(IServiceCollection serviceCollection)
   at Jellyfin.Server.Program.StartApp(StartupOptions options)
   at Jellyfin.Server.Program.Main(String[] args)
   at Jellyfin.Server.Program.<Main>(String[] args)
Aborted (core dumped)
```

I cannot even build the latest master branch:

```
  Installing Serilog 2.7.1.
  Installing Serilog 2.5.0.
  Installing Serilog 2.6.0.
  Installing Microsoft.Extensions.DependencyInjection 2.0.0.
  Installing Microsoft.Extensions.Options.ConfigurationExtensions 2.0.0.
  Installing Serilog.Extensions.Logging 2.0.0.
  Installing Microsoft.AspNetCore.Hosting.Abstractions 2.0.0.
  Installing Microsoft.Extensions.Logging 2.0.0.
  Installing runtime.linux-x64.Microsoft.NETCore.DotNetAppHost 2.1.8.
  Installing runtime.linux-x64.Microsoft.NETCore.DotNetHostResolver 2.1.8.
  Installing runtime.linux-x64.Microsoft.NETCore.DotNetHostPolicy 2.1.8.
  Installing runtime.linux-x64.Microsoft.NETCore.App 2.1.8.
  Generating MSBuild file /repo/Jellyfin.Server/obj/Jellyfin.Server.csproj.nuget.g.props.
  Generating MSBuild file /repo/Jellyfin.Server/obj/Jellyfin.Server.csproj.nuget.g.targets.
  Restore completed in 6.66 sec for /repo/Jellyfin.Server/Jellyfin.Server.csproj.
  Restoring packages for /repo/MediaBrowser.Api/MediaBrowser.Api.csproj...
  Generating MSBuild file /repo/MediaBrowser.Api/obj/MediaBrowser.Api.csproj.nuget.g.props.
  Generating MSBuild file /repo/MediaBrowser.Api/obj/MediaBrowser.Api.csproj.nuget.g.targets.
  Restore completed in 9.49 ms for /repo/MediaBrowser.Api/MediaBrowser.Api.csproj.
  Restoring packages for /repo/MediaBrowser.Common/MediaBrowser.Common.csproj...
  Generating MSBuild file /repo/MediaBrowser.Common/obj/MediaBrowser.Common.csproj.nuget.g.props.
  Generating MSBuild file /repo/MediaBrowser.Common/obj/MediaBrowser.Common.csproj.nuget.g.targets.
  Restore completed in 8.8 ms for /repo/MediaBrowser.Common/MediaBrowser.Common.csproj.
  Restoring packages for /repo/MediaBrowser.Controller/MediaBrowser.Controller.csproj...
  Generating MSBuild file /repo/MediaBrowser.Controller/obj/MediaBrowser.Controller.csproj.nuget.g.props.
  Generating MSBuild file /repo/MediaBrowser.Controller/obj/MediaBrowser.Controller.csproj.nuget.g.targets.
  Restore completed in 9.11 ms for /repo/MediaBrowser.Controller/MediaBrowser.Controller.csproj.
  Restoring packages for /repo/MediaBrowser.LocalMetadata/MediaBrowser.LocalMetadata.csproj...
  Generating MSBuild file /repo/MediaBrowser.LocalMetadata/obj/MediaBrowser.LocalMetadata.csproj.nuget.g.props.
  Generating MSBuild file /repo/MediaBrowser.LocalMetadata/obj/MediaBrowser.LocalMetadata.csproj.nuget.g.targets.
  Restore completed in 10.6 ms for /repo/MediaBrowser.LocalMetadata/MediaBrowser.LocalMetadata.csproj.
  Restoring packages for /repo/MediaBrowser.MediaEncoding/MediaBrowser.MediaEncoding.csproj...
  Generating MSBuild file /repo/MediaBrowser.MediaEncoding/obj/MediaBrowser.MediaEncoding.csproj.nuget.g.props.
  Generating MSBuild file /repo/MediaBrowser.MediaEncoding/obj/MediaBrowser.MediaEncoding.csproj.nuget.g.targets.
  Restore completed in 10.87 ms for /repo/MediaBrowser.MediaEncoding/MediaBrowser.MediaEncoding.csproj.
  Restoring packages for /repo/MediaBrowser.Model/MediaBrowser.Model.csproj...
  Generating MSBuild file /repo/MediaBrowser.Model/obj/MediaBrowser.Model.csproj.nuget.g.props.
  Generating MSBuild file /repo/MediaBrowser.Model/obj/MediaBrowser.Model.csproj.nuget.g.targets.
  Restore completed in 8.84 ms for /repo/MediaBrowser.Model/MediaBrowser.Model.csproj.
  Restoring packages for /repo/MediaBrowser.Providers/MediaBrowser.Providers.csproj...
  Generating MSBuild file /repo/MediaBrowser.Providers/obj/MediaBrowser.Providers.csproj.nuget.g.props.
  Generating MSBuild file /repo/MediaBrowser.Providers/obj/MediaBrowser.Providers.csproj.nuget.g.targets.
  Restore completed in 72.74 ms for /repo/MediaBrowser.Providers/MediaBrowser.Providers.csproj.
  Restoring packages for /repo/MediaBrowser.WebDashboard/MediaBrowser.WebDashboard.csproj...
  Generating MSBuild file /repo/MediaBrowser.WebDashboard/obj/MediaBrowser.WebDashboard.csproj.nuget.g.props.
  Generating MSBuild file /repo/MediaBrowser.WebDashboard/obj/MediaBrowser.WebDashboard.csproj.nuget.g.targets.
  Restore completed in 12.8 ms for /repo/MediaBrowser.WebDashboard/MediaBrowser.WebDashboard.csproj.
  Restoring packages for /repo/MediaBrowser.XbmcMetadata/MediaBrowser.XbmcMetadata.csproj...
  Generating MSBuild file /repo/MediaBrowser.XbmcMetadata/obj/MediaBrowser.XbmcMetadata.csproj.nuget.g.props.
  Generating MSBuild file /repo/MediaBrowser.XbmcMetadata/obj/MediaBrowser.XbmcMetadata.csproj.nuget.g.targets.
  Restore completed in 9.37 ms for /repo/MediaBrowser.XbmcMetadata/MediaBrowser.XbmcMetadata.csproj.
  Restoring packages for /repo/Mono.Nat/Mono.Nat.csproj...
  Generating MSBuild file /repo/Mono.Nat/obj/Mono.Nat.csproj.nuget.g.props.
  Generating MSBuild file /repo/Mono.Nat/obj/Mono.Nat.csproj.nuget.g.targets.
  Restore completed in 10.01 ms for /repo/Mono.Nat/Mono.Nat.csproj.
  Restoring packages for /repo/RSSDP/RSSDP.csproj...
  Generating MSBuild file /repo/RSSDP/obj/RSSDP.csproj.nuget.g.props.
  Generating MSBuild file /repo/RSSDP/obj/RSSDP.csproj.nuget.g.targets.
  Restore completed in 9.24 ms for /repo/RSSDP/RSSDP.csproj.
  MediaBrowser.Model -> /repo/MediaBrowser.Model/bin/Release/netstandard2.0/MediaBrowser.Model.dll
  MediaBrowser.Common -> /repo/MediaBrowser.Common/bin/Release/netstandard2.0/MediaBrowser.Common.dll
  MediaBrowser.Controller -> /repo/MediaBrowser.Controller/bin/Release/netstandard2.0/MediaBrowser.Controller.dll
  Emby.Drawing -> /repo/Emby.Drawing/bin/Release/netstandard2.0/Emby.Drawing.dll
  IsoMounter -> /repo/Emby.IsoMounting/IsoMounter/bin/Release/netstandard2.0/IsoMounter.dll
  Emby.Naming -> /repo/Emby.Naming/bin/Release/netstandard2.0/Emby.Naming.dll
  Emby.Notifications -> /repo/Emby.Notifications/bin/Release/netstandard2.0/Emby.Notifications.dll
  DvdLib -> /repo/DvdLib/bin/Release/netstandard2.0/DvdLib.dll
  MediaBrowser.Providers -> /repo/MediaBrowser.Providers/bin/Release/netstandard2.0/MediaBrowser.Providers.dll
  MediaBrowser.WebDashboard -> /repo/MediaBrowser.WebDashboard/bin/Release/netstandard2.0/MediaBrowser.WebDashboard.dll
  MediaBrowser.XbmcMetadata -> /repo/MediaBrowser.XbmcMetadata/bin/Release/netstandard2.0/MediaBrowser.XbmcMetadata.dll
  RSSDP -> /repo/RSSDP/bin/Release/netstandard2.0/RSSDP.dll
  Emby.Dlna -> /repo/Emby.Dlna/bin/Release/netstandard2.0/Emby.Dlna.dll
  Mono.Nat -> /repo/Mono.Nat/bin/Release/netstandard2.0/Mono.Nat.dll
  MediaBrowser.Api -> /repo/MediaBrowser.Api/bin/Release/netstandard2.0/MediaBrowser.Api.dll
  MediaBrowser.LocalMetadata -> /repo/MediaBrowser.LocalMetadata/bin/Release/netstandard2.0/MediaBrowser.LocalMetadata.dll
  Emby.Photos -> /repo/Emby.Photos/bin/Release/netstandard2.0/Emby.Photos.dll
  Emby.XmlTv -> /repo/Emby.XmlTv/Emby.XmlTv/bin/Release/netstandard2.0/Emby.XmlTv.dll
  BDInfo -> /repo/BDInfo/bin/Release/netstandard2.0/BDInfo.dll
  MediaBrowser.MediaEncoding -> /repo/MediaBrowser.MediaEncoding/bin/Release/netstandard2.0/MediaBrowser.MediaEncoding.dll
Library/DefaultPasswordResetProvider.cs(56,47): error CS1011: Empty character literal [/repo/Emby.Server.Implementations/Emby.Server.Implementations.csproj]
Library/DefaultPasswordResetProvider.cs(56,75): error CS1011: Empty character literal [/repo/Emby.Server.Implementations/Emby.Server.Implementations.csproj]
  Jellyfin.Drawing.Skia -> /repo/Jellyfin.Drawing.Skia/bin/Release/netstandard2.0/Jellyfin.Drawing.Skia.dll
The command '/bin/sh -c bash -c "source deployment/common.build.sh &&     build_jellyfin Jellyfin.Server Release linux-x64 /jellyfin"' returned a non-zero code: 1

```

--- a/Emby.Server.Implementations/Library/DefaultPasswordResetProvider.cs
+++ b/Emby.Server.Implementations/Library/DefaultPasswordResetProvider.cs
@@ -53,7 +53,7 @@ namespace Emby.Server.Implementations.Library
                 {
                     File.Delete(resetfile);
                 }
-                else if (spr.Pin.Replace('-', '').Equals(pin.Replace('-', ''), StringComparison.InvariantCultureIgnoreCase))
+                else if (spr.Pin.Replace("-", "").Equals(pin.Replace("-", ""), StringComparison.InvariantCultureIgnoreCase))
                 {
                     var resetUser = _userManager.GetUserByName(spr.UserName);
                     if (resetUser == null)

